public/coffee/ide/editor/directives/aceEditor/auto-complete/AutoCompleteManager.coffee

```diff
@@ -122,6 +125,9 @@ define [
 			range = new Range(end.row, 0, end.row, end.column)
 			lineUpToCursor = @editor.getSession().getTextRange(range)
 			commandFragment = getLastCommandFragment(lineUpToCursor)
+			commandName = getCommandNameFromFragment(commandFragment)
+			if commandName in ['begin', 'end']
```

Comment:
Nope, autocomplete triggers correctly when typing out the initial `\begin`, will put a gif in the ticket to demo